engine/crates/engine-v2/src/engine/trusted_documents.rs

```diff
@@ -28,7 +27,6 @@ pub(crate) struct OperationDocument<'a> {
 impl<'ctx, R: Runtime> PreExecutionContext<'ctx, R> {
     /// Determines what document should be used for the request and provides an appropriate cache
     /// key for the operation cache and as a fallback a future to load said document.
-    #[instrument(skip_all)]
```

Comment:
I think we should rather intrusment`handle_trusted_document_query` & `handle_apq` rather than the implementations here IMHO, contrary to hooks when there's no ambiguity of whether it's implemented or not.

Reply:
but up to you 🤷